Add js_type config option (#59)

Thomva opened the pull request:
I have added a `js_type` to the config.

Thomva committed:
Add js_type config option (#59)

mrtimbrook commented:
Thanks for doing this. Is there a way we can work the type into the `assets` array so we're not setting the same type for all assets? It would need to support both `array` and `string`.

Thomva committed:
Merge branch '1.x' of github.com:area17/blast into 1.x

Thomva committed:
Merge branch '1.x' of github.com:area17/blast into 1.x

Thomva committed:
Separate js asset types & update docs (area17#59)

Thomva commented:
Hi, sorry for taking so long to respond. I have updated the code. It now checks if the js asset has a `type`, and if so, it uses it in the script tag. I have also updated the README. Can this PR be reopened, or do I create a new one?
